src/rocdecode/roc_decoder.cpp

```diff
@@ -172,7 +172,20 @@ rocDecStatus RocDecoder::GetVideoFrame(int pic_idx, void *dev_mem_ptr[3], uint32
     return rocdec_status;
 }
 
-rocDecStatus RocDecoder::ReleaseVideoFrame(int pic_idx) {
+rocDecStatus RocDecoder::ReleaseVideoFrame(void *parser_hdl, int pic_idx) {
```

Comment:
As I mentioned above, we can release a frame associated with `pic_idx` directly from the utility class by calling the new API` rocDecParserReleaseFrame`. Please remove this `ReleaseVideoFrame` function from here.